source/operations/install-deploy-manage/deploy-operator-helm.rst

```diff
@@ -0,0 +1,316 @@
+                        runAsGroup: 1000
+                        runAsNonRoot: true
+                        runAsUser: 1000
+                      terminationMessagePath: /dev/termination-log
+                      terminationMessagePolicy: File
+                  dnsPolicy: ClusterFirst
+                  restartPolicy: Always
+                  schedulerName: default-scheduler
+                  securityContext:
+                    fsGroup: 1000
+                    runAsGroup: 1000
+                    runAsNonRoot: true
+                    runAsUser: 1000
+                  serviceAccount: minio-operator
+                  serviceAccountName: minio-operator
+                  terminationGracePeriodSeconds: 30
+            status:
+              conditions:
+                - lastTransitionTime: "2023-05-11T14:57:43Z"
+                  lastUpdateTime: "2023-05-11T14:57:43Z"
+                  message: Deployment does not have minimum availability.
+                  reason: MinimumReplicasUnavailable
+                  status: "False"
+                  type: Available
+                - lastTransitionTime: "2023-05-11T14:57:43Z"
+                  lastUpdateTime: "2023-05-11T14:57:44Z"
+                  message: ReplicaSet "minio-operator-674cf5cf78" is progressing.
+                  reason: ReplicaSetUpdated
+                  status: "True"
+                  type: Progressing
+              observedGeneration: 1
+              replicas: 2
+              unavailableReplicas: 2
+              updatedReplicas: 2
+		     
+      ``console-secret.yaml``:
+
+      Create a ``console-secret.yaml`` file with the following contents:
+
+      .. code-block:: shell
+         :class: copyable
+
+         apiVersion: v1
+         kind: Secret
+         metadata:
+           name: console-sa-secret
+           namespace: minio-operator
+           annotations:
+             kubernetes.io/service-account.name: console-sa
+         type: kubernetes.io/service-account-token
+
+   #. ``kubectl apply`` the YAML files to apply the configuration to your deployment:
+
+      .. code-block:: shell
+         :class: copyable
+
+         kubectl apply -f service.yaml
+         kubectl apply -f operator.yaml
+         kubectl apply -f console-secret.yaml
+
+#. Connect to the Operator Console
+
+   .. include:: /includes/common/common-install-operator-kubectl-validate-open-console.rst
+
+
+Deploy a Tenant
+~~~~~~~~~~~~~~~
+
+You can deploy a MinIO tenant using either the :ref:`Operator Console <minio-operator-console>` or Helm.
+To deploy a tenant with the Console, see :ref:`Deploy and Manage MinIO Tenants <minio-installation>`.
+
+To deploy a tenant with Helm:
+
+#. The following Helm command creates a MinIO tenant:
+
+   .. code-block:: shell
+      :class: copyable
+
+      helm install \
+      --namespace tenant-ns \
+      --create-namespace \
+      tenant-ns tenant-|operator-version-stable|.tgz
+
+#. Expose the tenant Console port
```

Comment:
This section is from `includes/k8s/quickstart.rst`. It could be made into an include, although there's already an awful lot of includes going on in the Operator pages.

Reply:
reworked this, as it presumes the plugin install method.